js/code.js

```diff
@@ -184,13 +166,62 @@ function doLogout()
 	window.location.href = "index.html";
 }
 
-function doGoToUpdate()
+function doGoToUpdateContact(string, el)
+{
+	let oldFirstName = "";
+	let oldLastName = "";
+	let oldEmail = "";
+	let oldPhone = "";
+	let UID="";
+
+	let contactElem = el.parentElement.parentElement;
+	oldFirstName = contactElem.querySelector('#firstName').innerHTML;
+	oldLastName = contactElem.querySelector('#lastName').innerHTML;
+	oldEmail = contactElem.getElementsByTagName('a')[0].innerHTML;
+	oldPhone = contactElem.querySelector('#phone').innerHTML;
+	UID = el.parentElement.id;
+
+
+	window.sessionStorage.setItem('contactFirstName',oldFirstName);
+	window.sessionStorage.setItem('contactLastName',oldLastName);
+	window.sessionStorage.setItem('contactEmail',oldEmail);
+	window.sessionStorage.setItem('contactPhone',oldPhone);
+	window.sessionStorage.setItem('contactUID',UID);
+
+	window.location.href = "updateContacts.html";
+
+}
+
+function contactsAutoFill()
 {
```

Comment:
Autofill was completed in order to increase User Accessibility. Also reminds user of which contact it is that they are updating. @cereusware